jpo-ode-svcs/src/test/java/us/dot/its/jpo/ode/coder/OdePsmDataCreatorHelperTest.java

```diff
@@ -2,29 +2,35 @@
 
 import static org.junit.Assert.assertNotNull;
 
+import java.nio.file.Files;
+import java.nio.file.Paths;
 import org.junit.Test;
-
 import us.dot.its.jpo.ode.model.OdePsmData;
-import us.dot.its.jpo.ode.util.XmlUtils.XmlUtilsException;
 
+/**
```

Comment:
Mainly checkstyle updates to this unit test. Moving sample data to dedicated files and javadocs.